src/article.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+function Article(id, type, sectionId, sectionName, webPublicationDate, webTitle, webUrl, apiUrl) {
+    this.id = id
+    this.type = type 
+    this.sectionId = sectionId
+    this.sectionName =sectionName
+    this.webPublicationDate = webPublicationDate
+    this.webTitle = webTitle
+    this.webUrl = webUrl
+    this.apiUrl = apiUrl
+
```

Comment:
Currently, the `Article` doesn't own its displaying. If you think back to one of the demos Alice did for the Notes app, we had the Notes component owning how its displaying itself.

The logic currently lives in https://github.com/makersacademy/news-summary-challenge/pull/276/files#diff-ec681c37e2090e8a8320da5dab21f4b7R16

but we could move this logic into a method attached to the Article component